<?xml version="1.0"?>
<launch>
  <node pkg="ball_picker" type="detector" name="detector" output="screen">
    <remap from="kinect_data" to="/camera/rgb/image_rect_color" />
    <remap from="detect_balls" to="/ball_detections" />
    <remap from="detect_hands" to="/hand_detections" />

    <param name="handclassifier" value="$(find ball_picker)/haarcascades/hand.xml" />
  </node>
</launch>
